.github/workflows/publish-to-test-pypi.yml

```
@@ -0,0 +1,43 @@
+name: Publish Python 🐍 distributions 📦 to TestPyPI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to TestPyPI
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/djangocms-versioning
```

This should probably be

```suggestion
      url: https:// test.pypi.org/p/djangocms-versioning
```